udt/fd.go

```diff
@@ -317,11 +322,11 @@ func closeSocket(sock C.UDTSOCKET) error {
 func dialFD(laddr, raddr *UDTAddr) (*udtFD, error) {
 
 	if raddr == nil {
-		return nil, &net.OpError{"dial", "udt", raddr, errors.New("invalid remote address")}
+		return nil, &net.OpError{Op: "dial", Net: "udt", Addr: raddr, Err: errors.New("invalid remote address")}
```

Comment:
not necessarily, but at some point since we last built this, another field was added to the struct.